### PR TITLE
TAJO-1450: TAJO-1450 Encapsulate Datum in Tuple

### PR DESCRIPTION
**Pull request by jihoonson**
Just rebased patch of @navis' work in #465.
His patch looks good to me. If any other objections, I'll commit.

[Review on Reviewable](https://reviewable.io/reviews/apache/tajo/596)

### CODE REVIEW

File: tajo-common/src/main/java/org/apache/tajo/storage/NullTuple.java [hunk 1]
```diff
@@ -117,6 +124,11 @@ public char getChar(int fieldId) {
   }
 
   @Override
+  public byte[] getTextBytes(int fieldId) {
+    return new byte[0];
```
Comment (jinossy) [inline tajo-common/src/main/java/org/apache/tajo/storage/NullTuple.java]: Can you change to static variable ?

File: tajo-common/src/main/java/org/apache/tajo/storage/VTuple.java [hunk 2]
```diff
@@ -132,6 +140,11 @@ public char getChar(int fieldId) {
 	}
 
   @Override
+  public byte[] getTextBytes(int fieldId) {
+    return values[fieldId].asTextBytes();
+  }
+
+  @Override
 	public short getInt2(int fieldId) {
```
Comment (jinossy) [inline tajo-common/src/main/java/org/apache/tajo/storage/VTuple.java]: please adjusting the indent

**Comment by jinossy** [comment]
@jihoonson 
Is the changes not related the following error ?
```
Failed tests: 
  TestValidators.testMinValidator:193 
Expected: is <true>
     but: was <false>
```

File: tajo-common/src/main/java/org/apache/tajo/storage/NullTuple.java (hunk 1 above)
Comment (jihoonson) [inline tajo-common/src/main/java/org/apache/tajo/storage/NullTuple.java]: Thanks. I've changed.

File: tajo-common/src/main/java/org/apache/tajo/storage/VTuple.java (hunk 2 above)
Comment (jihoonson) [inline tajo-common/src/main/java/org/apache/tajo/storage/VTuple.java]: Thanks. I've adjusted indent.

**Comment by jihoonson** [comment]
@jinossy, thanks for your review. That error seems not related to this patch. It would be better to investigate in another issue, I think.

**Comment by jinossy** [comment]
@jihoonson
I open the issue to TAJO-1635 and the patch looks good to me.
Here is my +1 
Thanks

**Comment by jihoonson** [comment]
Thanks! I'll commit shortly.

**Pull request closed without merging**